### PR TITLE
Don't fail the first time if there's nothing to clean up.

### PR DESCRIPTION
**Pull request by lukemarsden**
Would use `xargs -r` but it won't work on your Mac.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,6 +39,6 @@ binaries:
 	@cp "$(BUILD)/kubectl" "./"
 
 clean:
-	@docker ps -a | awk '$$2 !~ /weaveworks/ && $$1 !~ /^CONTAINER$$/ { print $$1 }' | xargs docker rm -f -v
-	@docker images | awk '$$1 ~ /none/ { print $$3 }' | xargs docker rmi -f
-	@docker images | awk '$$1 ~ /hyperquick$$/ && $$2 ~ /^(node|master)-v/ { print $$3 }' | xargs docker rmi -f
+	@docker ps -a | awk '$$2 !~ /weaveworks/ && $$1 !~ /^CONTAINER$$/ { print $$1 }' | xargs docker rm -f -v || true
```
Comment (errordeveloper) [inline Makefile]: `-@docker ...` is what you should do in make

**Comment by errordeveloper** [comment]
Ah, so GNU xargs returns an error if there is nothing on stdin?

**Comment by lukemarsden** [comment]
> Ah, so GNU xargs returns an error if there is nothing on stdin?

No, non-GNU xargs doesn't support -r, which would allow us to not run `docker rm` if there are no args for it...

**Comment by lukemarsden** [comment]
Fixed and merged master, this is ready for merge as well I think.

**Pull request merged**